packages/InlineEditing/stories/InlineEditing.stories.mdx

```diff
@@ -0,0 +1,278 @@
+      value={value} 
+      onClick={handleClick} 
+      isEditing={isEditing} 
+      onSubmit={handleSubmit} 
+    />
+  )
+}
+```
+
+  </Block>
+</Blocks>
+
+The previous example is a simple attempt to create a InlineEditng Input, to have it fully functionable will still required more work with the keyboard interactions,
+add focus onClick, cancel it onKeyUp === "Escape", etc.
+
+An because wiring a fully functionally InlineEditing component can be tedious and tricky to make it work properly, we currently are providing three components ready to use:
+
+- Input type text
+- TextArea
+- ListBox
+
+## Available InlineEditing Components
+
+### Input
+
+<Blocks>
+  <Block>
+    <InputStory />
+  </Block>
+  <Block>
+
+```jsx
+import Input from "@paprika/inline-editing/lib/Input";
+
+const [isEditing, setIsEditing] = React.useState(false);
+
+<Textarea
+  value="your value"
+  onSubmit={handleSubmit}
+  onEditingClose={() => setIsEditing(false)}
+  onEditingOpen={() => setIsEditing(true)}
+  isEditing={isEditing}
+/>;
+```
+
+  </Block>
+</Blocks>
+
+### TextArea
+
+<Blocks>
+  <Block>
+    <TextAreaStory />
+  </Block>
+  <Block>
+
+```jsx
+import TextArea from "@paprika/inline-editing/lib/Textarea";
+
+const [isEditing, setIsEditing] = React.useState(false);
+
+<Textarea
+  value="your value"
+  onSubmit={handleSubmit}
+  onEditingClose={() => setIsEditing(false)}
+  onEditingOpen={() => setIsEditing(true)}
+  isEditing={isEditing}
+/>;
+```
+
+  </Block>
+</Blocks>
+
+### ListBox
+
+<Blocks>
+  <Block>
+    <ListBoxSingleStory />
+    <div style={{ width: "100%", height: "44px" }}></div>
+    <ListBoxMultipleStory />
+  </Block>
+  <Block>
+
+```jsx
+import ListBox from "@paprika/inline-editing/lib/ListBox";
+
+const [isEditing, setIsEditing] = React.useState(false);
+
+/** 
+  Be aware that ListBox isMulti and the single select option have a different signature for the onChange and onEditing event 
+  isMulti={true} ([indexes], options, selectedIndex, {options})
+  isMulti={false} ([indexes], options, selectedIndex, {options})
+*/
+<ListBox
+  isEditing={isEditing}
+  onChange={() => {}}
+  onClose={() => {}}
+  onEditing={() => {}}
+  onSubmit={handleSubmit}
+  value={value}
+>
+  {someOptions.map(subscription => (
+    <ListBox.Option value={subscription} isSelected={subscription === value} key={subscription}>
+      {subscription}
+    </ListBox.Option>
+  ))}
+</ListBox>;
+```
+
+  </Block>
+</Blocks>
+
+## onSubmit vs onChange
+
+The inline-editing components provide two different callbacks onChange and onSubmit
+
+- `onChange` Will fire each time the component get an interaction from the user. (typing, selecting, etc)
+- `onSubmit` Will occurs only after the user either click `enter` (for certain component) or click outside of the editing state (onBlur)
+
+**NOTE:** if the user clicks `Esc` the onSubmit event will not fire. We aren't providing a `onEscape` callback neither, therefore would be hard to rollback an interaction
```

Comment:
```suggestion
**NOTE:** if the user clicks `Esc` the onSubmit event will not fire. We aren't providing an `onEscape` callback either, therefore it would be hard to rollback an interaction
```